generator/templates/Authorisation/src/api/endpoints/authorisation/register.js

```diff
@@ -1,22 +1,24 @@
 import { post } from '../../implementation/app';
 
-function register(email, name) {
-  return post('registration', {
+function verify(token, email, password, passwordConfirmation) {
+  return post('registration/verify', {
+    token,
     email,
-    name,
+    password,
+    passwordConfirmation,
   });
 }
 
-function verify(token, email, password, passwordConfirmation) {
-  return post('registration/verify', {
-    token: token,
+function acceptInvitation(email, token, password, passwordConfirmation) {
+  return post('invitation/accept', {
     email: email,
     password: password,
-    password_confirmation: passwordConfirmation,
+    passwordConfirmation,
```

Comment:
Gaat dit wel goed? Hoef je hier geen `key` aan te geven?

Reply:
Nope, dat hoeft sinds ES6 niet meer. De feature heet 'enhanced object literals' en het komt erop neer dat als de key hetzelfde is als de naam van de variabele, dan hoef je, in een object {...}, alleen de variabele in te geven. 
 
Bijvoorbeeld:
```javascript
const name = 'Jens'
console.log({ name})
```

Geeft als het goed de output
> { name: 'Jens' }

Trouwens, dit is echt een super handige manier om je variabelen te inspecteren, bijvoorbeeld console.log({ name, age, gender, employer }). Zo laat ie mooi alles zien en laat ie ook zien welke waarde bij welke key hoort.

Zie eventueel mbt 'enhanced object literals':
https://dev.to/sarah_chima/enhanced-object-literals-in-es6-a9d

En voor een overzicht van ES features (super behulpzaam, en sommige maken je code echt veel korter, minder repetitief en leesbaarder):
https://medium.com/@madasamy/javascript-brief-history-and-ecmascript-es6-es7-es8-features-673973394df4